common/test-framework/org.jboss.tools.common.reddeer/build.properties

```diff
@@ -0,0 +1,4 @@
+source.. = src/
+output.. = bin/
+bin.includes = META-INF/,\
+               .
```

Comment:
You're missing src.includes directives. See https://wiki.eclipse.org/Minerva#Source step 3.